dpkg/usr.bin.tox-node

```diff
@@ -0,0 +1,15 @@
+#include <tunables/global>
+
+/usr/bin/tox-node {
+  #include <abstractions/base>
+  #include <abstractions/nameservice>
+
+  /etc/tox-node/config.yml r,
+  /sys/fs/cgroup/cpu,cpuacct/cpu.cfs_quota_us r,
```

Comment:
Why is this needed?

Reply:
I did profiling and this the result of correct working daemon. Not sure _why_ it requires this access. On the other hand read-only access to cgroups should be safe.